nautiluszim/scraper.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         self.keep_build_dir = keep_build_dir
 
         self.build_dir = self.output_dir.joinpath("build")
+        self.tempfiles_dir = self.build_dir.joinpath("build")
```

Comment:
No I meant to use the `tempfile` module. You can create a `tempfile.NamedTemporaryFile` (inside `build_dir`) for each file to download.